ios/RCTConvert+BitmovinPlayer.swift

```diff
@@ -335,33 +335,62 @@ extension RCTConvert {
         }
     }
 
+    /**
+     Utility method to get a `DrmConfig` from a JS object.
+     - Parameter json: JS object
+     - Returns: The generated `DrmConfig` object
+     */
+    static func drmConfig(_ json: Any?) -> (fairplay: FairplayConfig?, widevine: WidevineConfig?) {
+        guard let json = json as? [String: Any?] else {
+            return (nil, nil)
+        }
+        return (
+            fairplay: RCTConvert.fairplayConfig(json["fairplay"]),
+            widevine: RCTConvert.widevineConfig(json["widevine"])
+        )
+    }
+
     /**
      Utility method to get a `FairplayConfig` from a JS object.
      - Parameter json: JS object
      - Returns: The generated `FairplayConfig` object
      */
     static func fairplayConfig(_ json: Any?) -> FairplayConfig? {
-        guard
-            let json = json as? [String: Any?],
-            let fairplayJson = json["fairplay"] as? [String: Any?],
-            let licenseURL = fairplayJson["licenseUrl"] as? String,
-            let certificateURL = fairplayJson["certificateUrl"] as? String
-        else {
+        guard let json = json as? [String: Any?],
+            let licenseURL = json["licenseUrl"] as? String,
+            let certificateURL = json["certificateUrl"] as? String else {
             return nil
         }
         let fairplayConfig = FairplayConfig(
             license: URL(string: licenseURL),
             certificateURL: URL(string: certificateURL)!
         )
-        if let licenseRequestHeaders = fairplayJson["licenseRequestHeaders"] as? [String: String] {
+        if let licenseRequestHeaders = json["licenseRequestHeaders"] as? [String: String] {
             fairplayConfig.licenseRequestHeaders = licenseRequestHeaders
         }
-        if let certificateRequestHeaders = fairplayJson["certificateRequestHeaders"] as? [String: String] {
+        if let certificateRequestHeaders = json["certificateRequestHeaders"] as? [String: String] {
             fairplayConfig.certificateRequestHeaders = certificateRequestHeaders
         }
         return fairplayConfig
     }
 
+    /**
+     Utility method to get a `WidevineConfig` from a JS object.
+     - Parameter json: JS object
+     - Returns: The generated `WidevineConfig` object
+     */
```

Comment:
Nit: IMO doc like this is not adding any value. No additional information besides what I see in the method signature. But fine for me too keep it as it aligns with the general practice in this file